### DISCUSSION TITLE
Editing map to remove crosswalk icon

### DISCUSSION THREAD

**Discussion by earthshaker77** [osmandapp/OsmAnd#16839] [Maps Related]
There is an area through which a crosswalk icon shows up. I have driven this road for over thirty years and there are no crosswalks anywhere close to the stretch of road. How can that be edited for removal?

**Comment by tommycrock** [comment]
Go to openstreetmap.org and click edit. You will have to create a login. Look at wiki.openstreetmap.org for hints and tips, ask on help.openstreetmap.org or ask here.

**Reply by earthshaker77** [reply]
I already have an account with open street map and it doesn't show there I will look at the wiki. Thanks!

**Reply by tommycrock** [reply]
Okay, sorry. Maybe you know this but crosswalks are often mapped just as a node on the highway, highway=crossing. Maybe post the way where it occurs or a lat/lon?

**Reply by scaidermern** [reply]
Please only remove it if you are really sure that the _data_ is wrong. It could be that the rendering is wrong instead and that the data is valid. We can check this if you provide more information, e.g. coordinates to the "crosswalk".

**Comment by earthshaker77** [comment]
I can do that on my next trip.

**Comment by earthshaker77** [comment]
Yeah, I am pretty sure it doesn't exist I drove through that area yesterday
it is in the middle of the country along a highway. There is about 3 miles
between where the icon shows up and where it disappears. but I will drive
out that way today and get the coordinates from where it starts and where
it ends, it is only a few miles from here.

On Thu, Apr 6, 2023 at 7:35 AM Alexander Heinlein ***@***.***>
wrote:

> Please only remove it if you are really sure that the *data* is wrong. It
> could be that the rendering is wrong instead and that the data is valid. We
> can check this if you provide more information, e.g. coordinates to the
> "crosswalk".
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/osmandapp/OsmAnd/discussions/16839#discussioncomment-5543261>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A62W4H6E56VJXPH6E7L54GDW722CNANCNFSM6AAAAAAWMLE7IY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Ryan Trullinger/Tru-Image Photography
KC0QNB Gothenburg Nebraska

**Reply by tommycrock** [reply]
Also crossings in OSM may be unmarked, i.e. just where a footway crosses a road, so it may be correct but not very obvious. Not sure how osmand handles those in terms of warnings

**Comment by earthshaker77** [comment]
Ok, thanks.

On Thu, Apr 6, 2023 at 9:34 AM tommycrock ***@***.***> wrote:

> Also crossings in OSM may be unmarked, i.e. just where a footway crosses a
> road, so it may be correct but not very obvious. Not sure how osmand
> handles those in terms of warnings
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/osmandapp/OsmAnd/discussions/16839#discussioncomment-5544415>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A62W4H2FFCSMT3JYGADZHTDW73IAZANCNFSM6AAAAAAWMLE7IY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Ryan Trullinger/Tru-Image Photography
KC0QNB Gothenburg Nebraska
